process/merge_requests/README.md

```diff
@@ -0,0 +1,104 @@
+   Set no label, this will be done by someone else.  
+   Set no assignee, unless you've been asked to.
+9. Press the "Submit merge request" button.
+
+This should take you to the page with the now created MR.  
+All that's necessary from here is to post a link to the MR over in the
+corresponding Redmine issue.
+
+## Title and Description guidelines
+The title should always look something like this:
+
+    #12345: Adjust constant in warpcore.yml to prevent overheating
+
+Notice the issue number at the start, including the `#` character. This text
+will automatically be linked to the Redmine issue, which makes it easy to access
+the discussion in Redmine.  
+Following the issue number should be general description of what the MR changes.
+
+The description should include the following:
+
+* What the MR changes, being a bit more descriptive than the title.
+* If there's something you're unsure of how it should be done, point this out so
+  the reviewer can help you out.
+* If necessary, include information that would help the reviewers understand
+  *why* you made certain changes. This information should already be present in
+  the form of [well written commit messages](../../style/git), but it doesn't
+  hurt including this in the MR description.
+* If there are parts of the changeset that can be ignored (you added a library
+  to vendor, resulting in thousands of lines of code), you should point this out
+  so the reviewer doesn't waste time sifting through vendor code.
+
+## Working with merge requests
+The process of working within MRs doesn't differ much from how review has been
+handled over in Redmine, with one major difference:
+
+If you make an adjustment (eg. to address a comment that was left by
+a reviewer) and push the resulting code, this will immediately be reflected in
+the MR. This way, the reviewer will always see the latest version of the branch
+in the MR.
+
+Additionally, commenting on specific lines in the code is very easy, so feel
+free to use the feature whenever it's applicable. To comment on a line, press
+the "Changes" tab near the top of the MR, then press a line number in one of the
+changed files to open up a comment box.
+
+Everywhere where you can leave a comment,
+[Gitlab flavoured Markdown](http://doc.gitlab.com/ce/markdown/markdown.html) is
+used. While it's helpful if you glance through the entire document, if you only
+take away one thing from that it, it should be that you can turn any text into
+monospaced by surrounding it with backticks (or grave accents, as they're also
+called: `` ` ``). If you're talking about `SomeClass.with.some_method`, things
+become easier to if you surround it with backticks.
+
+## Closing a merge requests
+
+When the reviewer considers the code ready to merge, he'll say so, and a label
+will be applied that indicates that the branch is ready to merge and deploy.
+Don't worry about the label, someone else will handle applying it.
```

Comment:
I chose this wording because to my knowledge the name of the label has already changed two times, maybe even three. Until it's set in stone (or at least a bit less volatile), I think we can rely on word of mouth, since the only people applying the labels are the select few that have admin rights.